mmv1/products/compute/GlobalForwardingRule.yaml

```diff
@@ -441,3 +454,11 @@ properties:
     name: allowPscGlobalAccess
     description: This is used in PSC consumer ForwardingRule to control whether the PSC endpoint can be accessed from another region.
     min_version: beta
+  - !ruby/object:Api::Type::Boolean
+    name: noAutomateDnsZone
+    description:
+      This is used in PSC consumer ForwardingRule to control whether it should try to auto-generate a DNS zone or not. 
+      Non-PSC forwarding rules do not use this field.
+    send_empty_value: true
+    immutable: true
```

Comment:
I was looking at [the docs for updating global forwarding rules](https://cloud.google.com/compute/docs/reference/rest/v1/globalForwardingRules/patch) and didn't see a note about being unable to update this field. What made you choose to set it as immutable here (and for the regional version of the resource)?

Reply:
@SarahFrench about this one, I see it as immutable because I can't update the field after it's created, the API show us it's possible to add the field in the body, but when trying to update I receive an error "Invalid value for field 'resource.noAutomateDnsZone': 'false'. Invalid field set in Private Service Connect Forwarding Rule. This field should not be set."
Another point, this field doesn't return on the GET, the reason I'm adding the ignore_read, another point in the doc of gcloud for forwarding rules update, there's no flag to update the field
https://cloud.google.com/sdk/gcloud/reference/compute/forwarding-rules/update.

But, I'm double checking with some Googler that works with this resource and will let you know.

Reply:
It's a shame that the docs don't reflect this field being unchangeable, though from what you described the decision to make this field immutable and ignored on read sounds correct.

I'm happy to approve this PR, though should we wait on what the Googler says?

Reply:
Hi @SarahFrench , thanks for the answers and the help.
While I was writing an answer to you I just received a message.
So the field can't be patched because it's just on creation time and unfortunately we don't have this information on the documentation, just figuring it out.

So I see we can move forward with the review
Thanks a lot